### PR TITLE
fixed the duplicate react version bug in rollup

### PR DESCRIPTION
**Pull request by Adi-11**
Just ignore the changes in other files, they are just format on save changes
Main changes are in: **rollup.config.js and package.json**
```
**in rollup.config.js**
external: [
    ...Object.keys(pkg.peerDependencies || {}),
    ...Object.keys(pkg.dependencies || {})
  ],
```
```
**in package.json**
"peerDependencies": {
    "react": "^17.0.2",
    "react-dom": "^17.0.2"
 },
and removed these two packages from "dependencies" 
```
Reference: https://hinammehra.medium.com/build-a-private-react-component-library-cra-rollup-material-ui-github-package-registry-1e14da93e790

### CODE REVIEW

**Comment by endurance21** [comment]
@Adi-11  do counter displays after this fix ?

**Comment by Adi-11** [comment]
Yes, it displayed perfectly!

File: package.json [hunk 1]
```diff
@@ -29,17 +28,22 @@
...
     "babel-core": "^6.26.3",
     "babel-plugin-syntax-dynamic-import": "^6.18.0",
     "babel-runtime": "^6.26.0",
     "rollup": "^2.40.0",
+    "rollup-plugin-paths": "^0.0.4",
     "rollup-plugin-sass": "^1.2.2",
     "rollup-plugin-typescript2": "^0.30.0"
   },
+  "peerDependencies": {
+    "react": "^17.0.2",
```
Comment (endurance21) [inline package.json:41]: why this 17.0.2 instead of 17.0.1 ?

**Comment by endurance21** [comment]
@Adi-11  thank you for your help

File: package.json (hunk 1 above)
Comment (Adi-11) [reply package.json:41]: It's just, I was trying to update react version to resolve the conflict before, I got the correct fixed to this issue.
I guess, now even if the version will be 17.0.1 thing will go smoothly..!!

**Pull request merged**